graspologic/align/orthogonal_procrustes.py

```diff
@@ -11,16 +11,16 @@ class OrthogonalProcrustes(BaseAlign):
     """
     Computes the matrix solution of the classical orthogonal Procrustes [1]_
     problem, which is that given two matrices `X` and `Y` of equal shape
```

Comment:
shouldn't all these all be double ticks as well? i guess one can argue that this should be math, but i was certainly referring to the inputs of .fit() here.

Reply:
but im by no means satoshi nakamoto and what i intended shouldnt be used as guidance for how things should be done now.

Reply:
comment re: double ticks applies throughout this file.

Reply:
or are arguments not double ticked, _unless_ they are also an attribute? i'm genuinely confused on the current convention.

Reply:
I interpreted *X* and *Y* as math.  If you wrote this and you intended to refer to parameters of ``fit()``, then they should be double ticked.

Reply:
I was referring to parameters (X, Y) and attributes (Q_) in this paragraph. but i'm totally okay with you changing this to math if you think that's more consistent with the rest.

Reply:
The whole point of standardizing these things is so that it is clear to the reader when a variable is referring to math vs. parameter vs. attribute etc.  I couldn't tell, when I first read this paragraph, whether it was math or a parameter (it didn't help that it was a parameter of the fit() method rather than the class itself).  So, since you intended these to refer to parameters, they should refer to parameters (right?).

Reply:
i meant: my point was that no matter what i did, you have more experience with the docs repo-side, and if we mostly use :math: in the similar place in other classes - then you should make it math, consistency within repo is of a higher priority.